src/fparser/tests/test_issue228.py

```diff
@@ -0,0 +1,102 @@
+# Modified work Copyright (c) 2017-2018 Science and Technology
```

Comment:
It's 2020 :-)

Reply:
I think in fact that you could just put these tests into src/fparser/common/tests/test_base_classes.py. (The use of a separate test file for each issue pre-dates our creation of the separate fparser project from f2py.)

Reply:
Done ... as in: moved it into the existing file.